packages/core/src/browser/keybinding.ts

```diff
@@ -261,6 +255,7 @@ export class KeybindingRegistry {
     containsKeybindingInScope(binding: Keybinding, scope = KeybindingScope.USER): boolean {
         const bindingKeySequence = this.resolveKeybinding(binding);
         const collisions = this.getKeySequenceCollisions(this.keymaps[scope], bindingKeySequence)
+            // eslint-disable-next-line deprecation/deprecation
```

Comment:
Why do we need such comments? If there are still usages of deprecated APIs. We still should see it rather than hide.

Reply:
It's build noise, and using such comments is actually an act of handling it. If we want to get a list of suppressed deprecations we can still search for `deprecation/deprecation`. The alternative would be to get rid of the deprecated API indeed, but here I simply had to suppress the warning to say "this is expected".

The line you attached your comment to handles an old API. I didn't know enough about this API to remove it, but if you tell me that we really don't want to keep supporting `context` and rather use `when` every time then I can remove it.

Reply:
I am fine with build noise if it is valid noise.

Reply:
Alternatively we should remove warnings completely from the build, but make sure that deprecation usages are highlighted as errors in the editor.

Reply:
The warnings should help when writing new code and having to decide what to do with said warning: Change the code or suppress because we must handle the deprecated but not yet removed API?

It is possible to remove the `eslint-disable` comments and only have the warnings show in the editor, but it adds complexity to the repo setup. And I think it is much easier to do like what is done in this PR, without losing anything.

Reply:
Regarding the code you highlighted, is it safe to remove this `context` attribute and only keep `when` on keybindings?